### PR TITLE
Wallet connect sanity

### PR DESCRIPTION
**Pull request by MPankajArun**

### CODE REVIEW

File: src/blockchain/web3/Loan.ts [hunk 1]
```diff
@@ -3,91 +3,141 @@ import { NumToBN } from '../utils';
...
+  loanRequest(
+    market: string,
+    commitment: string,
+    loanAmount: number,
+    loanDecimal: number,
+    collateralMarket: string,
+    collateralAmount: number,
+    collateralDecimal: number
+  ) {
+    debugger
```
Comment (Shrutika-hashstack) [inline src/blockchain/web3/Loan.ts]: remove debugger

File: src/blockchain/web3/Loan.ts [hunk 2]
```diff
@@ -3,91 +3,141 @@ import { NumToBN } from '../utils';
...
+    let loanAmountToSend = NumToBN(loanAmount, loanDecimal)
+    let collateralAmountToSend = NumToBN(collateralAmount, collateralDecimal)
```
Comment (Shrutika-hashstack) [inline src/blockchain/web3/Loan.ts]: why let use const

File: src/components/layout/Header.tsx [hunk 3]
```diff
@@ -32,7 +27,8 @@ const Header = () => {
 
   async function handleGetToken (event: any) {
     try {
-      await wrapper?.getFaucetInstance().getTokens(event.target.textContent);
+      const tx = await wrapper?.getFaucetInstance().getTokens(event.target.textContent);
+      onSuccessCallback(tx.events);
     } catch (error) {
       console.log(error);
```
Comment (Shrutika-hashstack) [inline src/components/layout/Header.tsx]: remove console.log

File: src/blockchain/utils.ts [hunk 4]
```diff
@@ -55,12 +55,12 @@ export const fixedSpecial = (num, n) => {
 
 export const BNtoNum = (value, decimal = 18) => {
```
Comment (Shrutika-hashstack) [inline src/blockchain/utils.ts]: type of function params, type of return value is not defined

File: src/blockchain/web3/Loan.ts (hunk 1 above)
Comment (MPankajArun) [reply src/blockchain/web3/Loan.ts]: Done

File: src/blockchain/web3/Loan.ts (hunk 2 above)
Comment (MPankajArun) [reply src/blockchain/web3/Loan.ts]: done

File: src/components/layout/Header.tsx (hunk 3 above)
Comment (MPankajArun) [reply src/components/layout/Header.tsx]: done

File: src/blockchain/utils.ts (hunk 4 above)
Comment (MPankajArun) [reply src/blockchain/utils.ts]: done

**Pull request merged**